src/plugin.js

```diff
@@ -72,6 +72,61 @@ export default function () {
         post() {
             if (context && context.isExtractMode() && potEntries.length) {
                 const poData = buildPotData(potEntries);
+
+                // Here we sort reference entries, this could be useful
+                // with conf. options extract.location: 'file' and sortByMsgid
+                // which simplifies merge of .po files from different
+                // branches of SCM such as git or mercurial.
+                const poEntries = poData.translations.context;
+                Object.keys(poEntries).forEach((k) => {
+                    const poEntry = poEntries[k];
+                    // poEntry has a form:
+                    // {
+                    //     msgid: 'message identifier',
+                    //     msgstr: 'translation string',
+                    //     comments: {
+                    //         reference: 'path/to/file.js:line_number\npath/to/other/file.js:line_number'
+                    //     }
+                    // }
+                    if (poEntry.comments && poEntry.comments.reference) {
+                        const cmp = (x, y) => {
```

Comment:
This comparator must be placed somewhere in utils or a separate file + it would be great to have a separate unit test for this comparator.